src/pybind/mgr/ssh/Vagrantfile

```diff
@@ -26,14 +26,15 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell" do |s|
     ssh_pub_key = File.readlines("#{Dir.home}/.ssh/id_rsa.pub").first.strip
     s.inline = "echo #{ssh_pub_key} >> /home/vagrant/.ssh/authorized_keys"
+    s.inline = "sudo cp -r /home/vagrant/.ssh /root/.ssh"
   end
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo yum install -y yum-utils
     sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
     sudo rpm --import 'https://download.ceph.com/keys/release.asc'
     curl -L https://shaman.ceph.com/api/repos/ceph/master/latest/centos/7/repo/ | sudo tee /etc/yum.repos.d/shaman.repo
-    sudo yum install -y ceph python36
-    sudo ln -s /usr/bin/python36 /usr/bin/python3
+    sudo yum install -y python36 podman ceph
```

Comment:
In the near future we mustn't rely on ceph being installed on the target systems. Currently this is still required though, as we execute ceph-volume outside of a container. (Fixing this in a separate PR)